src/Avalonia.Visuals/Rendering/ImmediateRenderer.cs

```diff
@@ -193,6 +201,12 @@ void IVisualBrushRenderer.RenderVisualBrush(IDrawingContextImpl context, IVisual
             Render(new DrawingContext(context), visual, visual.Bounds);
         }
 
+        internal static void Render(IVisual visual, DrawingContext context, bool updateTransformedBounds)
```

Comment:
One of our unit tests relies on updating `TransformedBounds` from a static `Render` method, so added this internal method for unit tests use.